### ISSUE TITLE
Method exposing the Initblk opcode

### ISSUE THREAD

**Issue by aL3891** [dotnet/corefx#5286]
Hello!
I'm looking at some optimizations in some unsafe code and i ran into an opcode called Initblk, http://stackoverflow.com/questions/1897555/what-is-the-equivalent-of-memset-in-c#_=_ and https://msdn.microsoft.com/en-us/library/system.reflection.emit.opcodes.initblk(v=vs.110).aspx

Is this now exposed somewhere in the core fx? if not, i'd love to help add it but i'm not sure what the best way to do it would be. The folks in the SO thread does it via emitting but i would be great to have built in, especially for --native down the line. 

Any thoughts on this are welcome!

**Comment by aL3891** [comment]
Having https://msdn.microsoft.com/en-us/library/system.reflection.emit.opcodes.cpblk(v=vs.110).aspx would also be great :)

**Comment by JonHanna** [comment]
https://github.com/JonHanna/Mnemosyne is a project of mine from a while ago that exposes `initblk` and `cpblk` in both aligned and unaligned versions with a bunch of reasonable overloads, along with avoiding `cpblk` in .NET on x86 where it doesn't actually perform that well (I use memcpy if available and the size copied is over 640 bytes and a highly unwound loop otherwise).

I haven't looked at how well it woks with DNX though I did recently put doing so on my to-do list. (Along with seeing if avoiding `cpblk` on x86 is still worth doing; apparently it improved with 4.6, that I've started experimenting with just last week).
